src/application/hooks/common/useIsomorphicLayoutEffect.ts

```diff
@@ -0,0 +1,4 @@
+import { useEffect, useLayoutEffect } from "react";
+
+export const useIsomorphicLayoutEffect =
```

Comment:
이 훅에는 네이밍의 뜻이 어떤건가욥?!

Reply:
Next에서 `Isomorphic`이란 것은 CSR, SSR 환경에서 모두 동작하는 코드를 뜻합니다!

SSR 환경에서 `useLayoutEffect`는 경고가 나기 때문에 useEffect를 대신 사용하고
CSR 환경에선 그대로 `useLayoutEffect`를 사용한다는 의미에용~

> Applications like this are known as universal applications because on the client and the server the same JavaScript runs. You can also hear the fancier word “isomorphic” being used, which means the same thing exactly.

- https://www.brainvire.com/blog/ssr-with-isomorphic-javascript/